app/templates/administrative-units/administrative-unit/local-involvements.hbs

```diff
@@ -0,0 +1,8 @@
+{{page-title "Betrokken lokale besturen"}}
+
+{{breadcrumb
+  "Betrokken lokale besturen"
+  route="administrative-units.administrative-unit.local-involvements"
```

Comment:
I think this also needs the model id of the administrative unit to prevent errors when child breadcrumbs are added.

https://github.com/lblod/frontend-contact-hub/blob/3b8e420b542bbb20763d026b4d72b4a9c060d1fc/app/templates/administrative-units/administrative-unit/ministers.hbs#L6